client/src/containers/home/StatisticFragment.js

```diff
@@ -10,17 +10,37 @@ const StatisticCard = (props) => (
   </Card>
 );
 
+const STATISTICS_URL = '/api/v1/statistics';
+
 export default () => {
+  const [state, setState] = React.useState({
+    peopleUnderRisk: '',
+    evaluatedBuildings: '',
+    consolidatedBuilding: '',
+  });
+  React.useEffect(() => {
+    fetch(STATISTICS_URL)
+      .then((res) => res.json())
+      .then(({ people_under_risk, evaluated_buildings, consolidated_buildings }) => {
```

Comment:
Snake case notation breaks eslint rules, if I go for the computed property notation (e.g: statistics['people_under_risk']) it still returns an error, suggesting dot notation.
I could disable the eslint rule for this line or we could change the overall eslint config 🤔

Reply:
Can't we just change their names with a _camelCase_ version? If we're going to have conflicts if we have changes in `peopleUnderRisk: peopleUnderRisk`I think we can have it named as `peopleUnderRisk: peopleUnderRiskState` or something similar 🤔

Reply:
There's no naming collisions, I just copied the names of the fields from the ticket in which the API was described - there were written in snake case, but if there's used a serializer or something that's great

Reply:
Fair enough. Probably will be using camelcase for the endpoint 🤔 
Either this or I'll be refactoring it 😅